mex/backend/merged/transform.py

```diff
@@ -28,8 +29,14 @@ def transform_graph_results_to_merged_item_search_response_facade(
         try:
             model = transform_search_result_to_model(result)
             model_dict = model.model_dump()
-            model_dict["entityType"] = f"Merged{model_dict['entityType']}"
-            items.append(model_dict)
+            # create a MergedModel class with the dictionary
+            model_dict["entityType"] = model_dict["entityType"].replace(
+                "Extracted", "Merged"
+            )
+            del model_dict["hadPrimarySource"]
```

Comment:
could just use `model_dump(exclude={"hadPrimarySource", "identifierInPrimarySource"})` above

Reply:
True! Done in: 3224603dfdadce084d23cb81619c1c195ba4c682